### ISSUE TITLE
Cannot upgrade from 5.2.0 to 5.3.0

### ISSUE THREAD

**Issue by adeem** [idnow/de.idnow.android#151]
Keep getting error about "no static method setTransactionToken"
The way to set the Token code is changed now?

**Comment by KraljSamo** [comment]
Not mentioned in the documentation, but `setTransactionToken` and `getTransactionToken` methods changed so that they dont accept `context` parameter anymore.

You have to change that:

`IDnowSDK.setTransactionToken(transactionToken, context);` ->` IDnowSDK.setTransactionToken(transactionToken);`
`IDnowSDK.getInstance().start(IDnowSDK.getTransactionToken(context));` -> `IDnowSDK.getInstance().start(IDnowSDK.getTransactionToken());`

**Comment by LuisMesquita** [comment]
I'm trying to update my SDK from 5.2.0 to 5.3.0 as well, and I'm facing the following error. Any idea of what could it be?
<img width="430" alt="Screen Shot 2022-12-12 at 7 30 43 PM" src="https://user-images.githubusercontent.com/12140099/207125640-0f22ffd4-4c78-4c44-9c50-87fceb50736f.png">

**Comment by suzukieng** [comment]
@LuisMesquita I have the same problem after upgrading from 5.1.7 to 5.3.0...
There's always something with this SDK, never boring

**Comment by LuisMesquita** [comment]
Yeah, Idnow suggested me to update to 6.1.3 directly. Maybe it works for you too.

**Comment by suzukieng** [comment]
I'm a bit hesitant to do that, as they don't even mention breaking changes in minor releases (like setTransactionToken no longer requiring Context). Testing with 5.2.0 looks ok, so I'm sticking with that for now.
